### ISSUE TITLE
Doesn't work with packer and LunarVim

### ISSUE THREAD

**Issue by unkindypie** [kndndrj/nvim-dbee#81]
Tried to install nvim-dbee to my LunarVim via packer by adding this configuration to my configs:
```
  {
    "kndndrj/nvim-dbee",
    requires = {
      "MunifTanjim/nui.nvim",
    },
    run = function()
      -- Install tries to automatically detect the install method.
      -- if it fails, try calling it with one of these parameters:
      --    "curl", "wget", "bitsadmin", "go"
      require("dbee").install()
    end,
    config = function()
      require("dbee").setup( --[[optional config]])
    end
  },

```

but got this error when installing the package:
![image](https://github.com/kndndrj/nvim-dbee/assets/34601252/d787b7a4-0bda-49d1-8cd8-caeebbe9fc5a)
And this error when starting LunarVim (which is weird as I don't use lazy, I use packer):
![image](https://github.com/kndndrj/nvim-dbee/assets/34601252/c073792e-5692-4b41-a675-69dcee996f89)

My versions:
NVIM v0.9.5
LunarVim 1.3

**Comment by slarse** [comment]
I encountered the same problem, there's already a PR that fixes it. See #78.

**Comment by kndndrj** [comment]
Fixed with #98
